frontend/packages/ux-editor/src/components/config/EditFormContainer.tsx

```diff
@@ -50,6 +46,8 @@ export const EditFormContainer = ({
   const textResources: ITextResource[] = useTextResourcesSelector<ITextResource[]>(
     textResourcesByLanguageSelector(DEFAULT_LANGUAGE),
   );
+  useLayoutSchemaQuery(); // Ensure we load the layout schemas so that component schemas can be loaded
```

Comment:
I think this call is done within `useComponentSchemaQuery` now, so no need to trigger this call explicitly here